pgmq-extension/sql/pgmq--1.4.1--1.4.2.sql

```diff
@@ -187,6 +204,70 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
+
+CREATE OR REPLACE FUNCTION pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN DEFAULT FALSE)
+RETURNS BOOLEAN AS $$
+DECLARE
+    qtable TEXT := pgmq.format_table_name(queue_name, 'q');
+    fq_qtable TEXT := 'pgmq.' || qtable;
+    atable TEXT := pgmq.format_table_name(queue_name, 'a');
+    fq_atable TEXT := 'pgmq.' || atable;
+BEGIN
+    EXECUTE FORMAT(
+        $QUERY$
+        ALTER EXTENSION pgmq DROP TABLE pgmq.%I
+        $QUERY$,
+        qtable
+    );
```

Comment:
Theses short calls can use single quotes, e.g.:

```suggestion
    EXECUTE FORMAT('ALTER EXTENSION pgmq DROP TABLE pgmq.%I', qtable);
```

Reply:
This is pre-existing implementation. Let's address these style and formatting issues in separate work so we can keep scope of this PR on resolving the SQL injection issues.